hugo/layouts/report.html

```diff
@@ -30,7 +30,16 @@
 
   <!-- Body of the report -->
   <div class="container">
+
     <div class="card-body">
+    <!-- Search again button that returns user to homepage -->
+    <p>
+      {{- /* Determine the search‑page URL based on environment */ -}}
+      {{ $searchHref := cond (eq .Params.environment "prod") "." "https://stage-drupal.ccaotest.com/model-valuation" }}
+      <a href="{{ $searchHref }}" class="text-decoration-none fw-semibold">
+        ⬅Search again
+      </a>
+    </p>
```

Comment:
I know this isn't a paragraph, I started with a `div` block but I couldn't get the alignment right unless I used the `p` block. Any suggestions on tag or class changes here?